rpcs3/Emu/SysCalls/FuncList.cpp

```diff
@@ -2560,297 +2560,47 @@ std::string SysCalls::GetFuncName(const u64 fid)
-	case 0x04ca5e6a: return "sceNpScoreRecordGameData";
-	case 0x0561448b: return "sceNpCommerceGetDataFlagAbort";
-	case 0x05af1cb8: return "sceNpBasicGetMatchingInvitationEntry";
-	case 0x05d65dff: return "sceNpScoreGetRankingByNpId";
-	case 0x0968aa36: return "sceNpManagerGetTicket";
-	case 0x14497465: return "sceNpMatchingQuickMatchGUI";
-	case 0x155de760: return "sceNpSignalingGetConnectionInfo";
-	case 0x166dcc11: return "sceNpLookupNpId";
-	case 0x1672170e: return "sceNpScoreRecordScore";
-	case 0x168a3117: return "sceNpBasicAddPlayersHistory";
-	case 0x168fcece: return "sceNpManagerGetAccountAge";
-	case 0x16f88a6f: return "sceNpManagerGetPsHandle";
-	case 0x1a2704f7: return "sceNpScoreWaitAsync";
-	case 0x1a3fcb69: return "sceNpCommerceGetSkuUserData";
-	case 0x1ae8a549: return "sceNpBasicAddBlockListEntry";
-	case 0x1fdb3ec2: return "sceNpLookupUserProfileWithAvatarSizeAsync";
-	case 0x21206642: return "sceNpScoreGetRankingByRangeAsync";
-	case 0x227f8763: return "sceNpScoreGetClansRankingByClanIdAsync";
-	case 0x259113b8: return "sceNpScoreDestroyTitleCtx";
-	case 0x260caedd: return "sceNpBasicGetFriendPresenceByNpId2";
-	case 0x2687a127: return "sceNpSignalingGetCtxOpt";
-	case 0x26b3bc94: return "sceNpMatchingGetResultGUI";
-	case 0x26f33146: return "sceNpCommerceGetProductCategoryStart";
-	case 0x2706eaa1: return "sceNpScoreSetPlayerCharacterId";
-	case 0x276c72b2: return "sceNpSignalingSetCtxOpt";
-	case 0x27c69eba: return "sceNpBasicAddFriend";
-	case 0x29dd45dc: return "sceNpScoreSetTimeout";
-	case 0x2a76895a: return "sceNpScoreGetClansRankingByClanId";
-	case 0x2ad7837d: return "sceNpMatchingAcceptInvitationGUI";
-	case 0x2be41ece: return "sceNpCommerceGetNumOfChildCategory";
-	case 0x2cd2a1af: return "sceNpScoreSanitizeCommentAsync";
-	case 0x2e1c5068: return "sceNpMatchingDestroyCtx";
-	case 0x2ecd48ed: return "sceNpDrmVerifyUpgradeLicense";
-	case 0x2f2c6b3e: return "sceNpProfileAbortGui";
-	case 0x2fccbfe0: return "sceNpLookupUserProfileWithAvatarSize";
-	case 0x30d1cbde: return "sceNpBasicGetMessageEntry";
-	case 0x32200389: return "sceNpManagerGetMyLanguages";
-	case 0x32c78a6a: return "sceNpBasicGetFriendPresenceByIndex";
-	case 0x32cf311f: return "sceNpScoreInit";
-	case 0x32febb4c: return "sceNpMatchingSearchJoinRoomGUI";
-	case 0x34cc0ca4: return "sceNpMatchingKickRoomMember";
-	case 0x34ce82a0: return "sceNpSignalingGetConnectionFromPeerAddress";
-	case 0x359642a6: return "sceNpCommerceGetCategoryDescription";
-	case 0x36d0c2c5: return "sceNpManagerGetAvatarUrl";
-	case 0x39a69619: return "sceNpCommerceGetSkuId";
-	case 0x3b02418d: return "sceNpScoreGetGameData";
-	case 0x3cc8588a: return "sceNpMatchingCreateRoomGUI";
-	case 0x3d1760dc: return "sceNpLookupAbortTransaction";
-	case 0x3db7914d: return "sceNpScoreGetRankingByNpIdAsync";
-	case 0x3f0808aa: return "sceNpBasicSetPresence";
-	case 0x3f195b3a: return "sceNpCommerceGetProductCategoryResult";
+	case 0x816c6a5f: return "_sceNpSysutilClientFree";
```

Comment:
What are you trying to change in FuncList.cpp? Please, stop it. It's not for it.